src/webBehavior.ts

```diff
@@ -173,45 +175,44 @@ module powerbi.extensibility.visual {
             if (!this.slicerSettings.general.forcedSelection) {
                 return;
             }
-            const isSelected: boolean = _.some(this.dataPoints, (dataPoint: ChicletSlicerDataPoint) => dataPoint.selected);
+
+            const isSelected: boolean = _.some(
+                this.dataPoints,
+                (dataPoint: ChicletSlicerDataPoint) => dataPoint.selected);
 
             if (!isSelected) {
                 for (let i: number = 0; i < this.dataPoints.length; i++) {
-                    let dataPoint: ChicletSlicerDataPoint = this.dataPoints[i];
+                    const dataPoint: ChicletSlicerDataPoint = this.dataPoints[i];
+
                     if (dataPoint.selectable && !dataPoint.filtered) {
                         this.selectionHandler.handleSelection(dataPoint, false);
+
                         this.saveSelection();
+
                         break;
                     }
                 }
             }
         }
 
         public loadSelection(): void {
-            let savedSelectionIds: string[] = this.slicerSettings.general.getSavedSelection();
+            const savedSelectionIds: ISelectionId[] = this.slicerSettings.general.getSavedSelection();
+
             if (savedSelectionIds.length) {
                 this.selectionHandler.handleClearSelection();
-                let selectedDataPoints = this.dataPoints.filter(d => savedSelectionIds.some(x => (d.identity as any).getKey() === x));
-                selectedDataPoints.forEach(x => this.selectionHandler.handleSelection(x, true));
-            }
-        }
 
-        public static getFilterFromSelectors(
-            selectedIds: ISelectionId[],
-            isSelectionModeInverted: boolean,
-            identityFields: ISQExpr | ISQExpr[]): ISemanticFilter {
-
-            let selectors: Selector[] = [],
-                filter: ISemanticFilter;
+                this.dataPoints
+                    .filter(dataPoint => {
+                        return savedSelectionIds.some((selectionId: ISelectionId) => {
+                            return (dataPoint.identity as any).getKey() === selectionId;
+                        });
+                    })
+                    .forEach((dataPoint: ChicletSlicerDataPoint) => {
```

Comment:
Yes, it can, but it might happen if user selects a lot of items and saves the report. I suppose that our users don't do so, because we haven't had any problems with it (I haven't seen any email about it).
We will analyze it deeper when we work on the Memory leaks task story.